### DISCUSSION TITLE
GINAN POD

### DISCUSSION THREAD

**Discussion by yiruite** [GeoscienceAustralia/ginan#100] [General]
Dear GINAN team:
       This is my first time to build and use your software, Thank you all for your contributions and open source!  
       After I successfully built from the files and videos, I found that there was no POD module,Is this module no longer public?
       I have another question for you. I tried to correct a set of data using the SSR of Galileo HAS. Can the orbit error analysis be carried out based on GINAN PEA?
       Thank you for your attentions!

**Comment by seballgeyer** [comment]
Thank you for reaching out and for your kind words about the GINAN software! We're glad to hear that you successfully built the software. Regarding your question, the POD (Precise Orbit Determination) module is now fully integrated within the PEA (Parameter Estimation Algorithms) module and offers similar functionalities than the original POD. This means you won’t find a separate POD module, as its functionality is now part of PEA.

This integration aims to streamline the workflow and enhance the overall efficiency of the software. By consolidating the functionalities, we reduce redundancy and simplify the process for users like yourself. However, we understand that this change might not be immediately apparent, especially for those who are used to the previous structure.

Regarding your interest in correcting data using the SSR of Galileo HAS, this is indeed an interesting and valuable topic. Currently, we don't have the resources to support this work extensively in the next six months. However, if you're willing to take the initiative, we can offer limited support to guide you through the process. Please feel free to reach out if you'd like to proceed with this.

**Reply by yiruite** [reply]
Dear Professor Sebastien:
       I am so grateful for your reply! Given the huge potential and widespread interest in open services such as PPP-B2b/HAS in precision targeting, I am very willing to learn more about the implementation of GINAN software with you, of course, on the basis of my learning and familiarity with this software as soon as possible. :-)
